openapi-specs/schema.json

```diff
@@ -2667,6 +2694,211 @@
         }
       }
     },
+    "/users/{user_id}": {
+      "get": {
+        "summary": "get info relevant to user, e.g. username, roles",
+        "operationId": "getUserInfo",
+        "x-serviceIds": [
+          "weaviate.users.id.get"
+        ],
+        "tags": [
+          "users"
+        ],
+        "parameters": [
+          {
+            "description": "user id",
+            "in": "path",
+            "name": "user_id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Info about the user",
+            "schema": {
+              "$ref": "#/definitions/UserInfoOnlyNames"
```

Comment:
Looking back at the return of `/users/own-info`, I think that should be named `UserOwnInfo` instead of `UserInfo`. Then this one can be named `UserInfo`. wdyt? Changing the names here shouldn't be a breaking change since it's only the shape of the data that matters in a REST request, not the data's name.

Reply:
Yep, good idea. I made a PR against 1.28 for the renaming to avoid merge conflicts: https://github.com/weaviate/weaviate/pull/7354 - will make the changes here after it has propagated

Reply:
done!